tgui/packages/tgui/interfaces/TextInputModal.tsx

```diff
@@ -15,6 +15,22 @@ type TextInputData = {
   title: string;
 };
 
+export const SanitizeMultiline = (toSanitize: string) => {
```

Comment:
Doesn't look very optimized, and I'd like if you used lowercase first letter for function names, but otherwise I approve.